docs/infra/website.md

```diff
@@ -0,0 +1,196 @@
+Website (fossrit.github.io)
+===========================
+
+This page documents the official FOSS@MAGIC website hosted on GitHub Pages.
+The website source code is found in [FOSSRIT/fossrit.github.io](https://github.com/FOSSRIT/fossrit.github.io "Official website for Free and Open Source Software @ RIT MAGIC Center and FOSS academia").
+
+
+## About
+
+* The site is built using [Jekyll](https://jekyllrb.com/), [Bootstrap](https://getbootstrap.com/) (via CDN), [Font Awesome](https://fontawesome.com/) (via CDN), & [GitHub Pages](https://help.github.com/categories/github-pages-basics/)
+* [PDF.js](https://mozilla.github.io/pdf.js/) is used for embedding PDFs in pages (via CDN)
+* "Pretty links" is turned on in the Jekyll configuration (see [`permalink: pretty`](https://github.com/FOSSRIT/fossrit.github.io/blob/master/_config.yml))
+* `github-pages` Gem used to simplify dependencies
+* The live site is updated by committing changes to the `master` branch
+
+_The site uses an undocumented feature of Jekyll where categories are defined using `category/_posts/` instead of `_posts/category/`._
+
+
+## How to update FOSS Hours time/place
+
+Edit [`_config.yml`](https://github.com/FOSSRIT/fossrit.github.io/blob/master/_config.yml) and update the settings `meeting-day`, `meeting-place`, and `meeting-time`.
+These settings are used across the website automatically.
+The meeting time and place are referenced in the `_config.yml` so they are updated in one place.
+
+Example:
+
+```yaml
+# --- meeting schedule settings ---
+
+# They appear exactly as typed. Follow the format already here when updating.
+meeting-day: Thursday
+meeting-place: MSS/71-3190 (conference room)
+meeting-time: 5:00PM - 7:00PM
+```
+
+
+## How to add content
+
+This section describes how to add new content to the FOSS@MAGIC website.
+
+Add a new `.md` file to one of the `_posts/` folders for a category.
+Store assets (a.k.a. pictures, PDFs, or other downloadable files) in a folder within that category (e.g. `projects/assets/`).
+Include the date in asset file names so it is clear what references it.
+
+Content categories include the following:
+
+* [Announcements](https://github.com/FOSSRIT/fossrit.github.io/tree/master/announcements)
+* [Events](https://github.com/FOSSRIT/fossrit.github.io/tree/master/events)
+* [Projects](https://github.com/FOSSRIT/fossrit.github.io/tree/master/projects)
+
+### Git work flow
+
+1. Fork the repository to your own Github account (go to [FOSSRIT/fossrit.github.io](https://github.com/FOSSRIT/fossrit.github.io) and click "fork")
+2. Clone repository locally to your workstation
+3. Create/add files
+4. Git commit
+5. Git push to your fork
+6. Open a pull request against `master` branch of FOSSRIT/fossrit.github.io repository
+7. When pull request is merged, site updates in a couple of minutes
```

Comment:
Minor grammar, needs articles ("a"/"an"/"the") for "pull request" &/or "site"

Reply:
I did this intentionally as a technical writing preference, but I wasn't consistent in that style. I'll follow up shortly.

Reply:
Addressed in commit b4a213117093c27d880b65cd6835d0bd46d14663.

Reply:
Having no articles here sounds a little strange since they're used throughout the rest of the document, but I'm not going to block on it.